packages/lit-dev-content/site/learn/index.html

```diff
@@ -0,0 +1,57 @@
+---
+title: Learn
+---
+
+{% extends 'default.html' %} {% block head %} {% include "social-tags.html" %}
+{% inlinecss "learn-catalog.css" %} {% endblock %} {% block content %}
+<script type="module" src="{{ site.baseurl }}/js/pages/learn.js"></script>
+
+<div id="hero">
+  <div id="hero-contents">
+    <h1>Learn</h1>
+    <p>
+      Explore a curated list of videos, articles, and tutorials tailored for all
+      experience levels. Unlock the full potential of Lit.
+    </p>
+  </div>
+</div>
+<section id="learn-catalog">
+  <div class="center">
+    <md-chip-set id="chips" aria-label="Filter content by:">
+      <md-filter-chip data-value="article" label="Articles" selected></md-filter-chip>
+      <md-filter-chip data-value="video" label="Videos" selected></md-filter-chip>
+      <md-filter-chip data-value="tutorial" label="Tutorials" selected></md-filter-chip>
+    </md-chip-set>
+    <div class="card-grid">
+      {% for content in learn %}
+      <a class="card" data-content-kind="{{ content.kind }}" href="{{ content.url }}">
+        <div class="card-header">
+          {% if content.kind == "video" %}
+          <img
```

Comment:
I think we should have an `<img>` tag for every card, but fall back to a default if something doesn't have one.

Right away I want to make a custom card for the Polymer article, and probably a default tutorial card.

Reply:
for making custom cards, can you put in a readme somewhere what the image size should be?

Reply:
I added it to the learn 11ty data file, near the thumbnail error message.